test/promptsource/test_augmentation.py

```diff
@@ -0,0 +1,19 @@
+import augmentation as aug
+
+CASE_AG_NEWS_1 = (
+    "{{text}}"
+    "\nIs this a piece of news regarding "
+    "{{\"world politics\"}}, {{\"sports\"}}, {{\"business\"}}, or {{\"science and technology\"}}? |||"
+    "\n{{ [\"World politics\", \"Sports\", \"Business\", \"Science and technology\"][label] }}"
+)
+
+
+def test_back_translation_strategy():
+    aug_tmpl = aug.back_translation_strategy(CASE_AG_NEWS_1)
+    assert aug_tmpl == CASE_AG_NEWS_1.replace(
+        "a piece of news regarding",
+        "a new one about"
+    ).replace(
+        ", or",
+        " or"
```

Comment:
Empirically, the reason why this augmentation is less than ideal ("a new one" and the missing comma) is probably due to the multiple constants and their long replacements (e.g., `x_cnst_1_0`).